assignment1/main.py

```diff
@@ -0,0 +1,56 @@
+print("")
+def first_prime_number(number):
+    def is_not_prime(number):
```

Comment:
In general it is bad to write functions that check for a negation. Instead please change this function to `is_prime`.